docs/parity-testing.md

```diff
@@ -0,0 +1,260 @@
+# Parity Testing
+
+Parity tests (also called snapshot tests) are a special form of integration tests that should verify and improve the correctness of LocalStack compared to AWS.
+
+Initially, the integration test is executed against AWS and collects responses of interest. Those responses are called "snapshots" and will be used later on to compare the results from AWS with the ones from LocalStack.
+Those responses aka "snapshots" are stored in a **snapshot.json** file.
+
+Once the snapshot is recorded, the test can be executed against LocalStack. During this “normal” test execution, the test runs against LocalStack and compares the LocalStack responses with the recorded content.
+
+In theory, every integration test can be converted to a parity conform snapshot test.
+
+This guide assumes you are already familiar with writing [integration tests](integration-tests) for LocalStack in general. 
+
+## How to write Parity tests
+
+In a nutshell, the necessary steps include:
+
+1.  Make sure that the test works against AWS. 
+
+    * Check out our [Integration Test Guide](integration-tests#running-integration-tests-against-aws) for tips on how run integration tests against AWS.
```

Comment:
I am pretty sure this link doesn't work as it should have the file ending (but I might be wrong?)...